build.ps1

```diff
@@ -49,6 +49,7 @@ foreach ($profile in $profiles) {
   New-Item -ItemType Directory -Force -Path ".\build_$profile\" | Out-Null
 
   & $conan.Path lock create "$PSScriptRoot\conanfile.py" --user=orbitdeps --channel=stable `
+    --build outdated `
```

Comment:
Maybe we could you
```suggestion
    --build=outdated `
```
as below for consistency? (and the same in `build.sh`)

Reply:
Good point. Will fix that!